src/dsymbol/makex.d

```diff
@@ -0,0 +1,63 @@
+module dsymbol.makex;
+
+version = safeAlloc;
+
+auto makeX(T, Allocator, A...)(auto ref Allocator alloc, auto ref A args)
```

Comment:
Why not simply redefine `make` and `Mallocator` depending on the allocator? That would make this PR have a minimal diff. For example:

 ```d
version(safeAlloc)
{
    import stdx.allocator.gc_allocator : GCAllocator;
    alias Mallocator = GCAllocator;
}
else
{
    import stdx.allocator.mallocator : Mallocator;
    alias Mallocator = stdx.allocator.mallocator.Mallocator;
}
```

And then redefined the `make` and `dispose` methods accordingly.

Reply:
That's ok with me.

Reply:
Furthermore, I could track down which Mallocator that results in the crash and hardcode that to always use GCAllocator for the time being. What do you say, @RazvanN7?

Reply:
I think that we should just use GCAllocator everywhere and apply the minimal changes for that. However, any solution works for me as long as we unblock phobos ASAP